zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGenerator.scala

```diff
@@ -38,7 +38,10 @@ class AQLGenerator extends IQLGenerator {
   }
 
   def parseCreate(create: CreateView, sourceSchema: AbstractSchema): String = {
-    val resultSchema = calcResultSchema(create.query, sourceSchema).asSchema
+    if (!sourceSchema.hasTimeField){
```

Comment:
since it won't work for `lookupSchema`, can we just change the signature from `sourceSchema: AbstractSchema` to `sourceSchema: Schema` ?